### PR TITLE
Add support for array of prince options

### PR DESCRIPTION
**Pull request by rsands2801**
Ability to pass in array to set document options for prince features

### CODE REVIEW

File: src/ApiWrapper.php [hunk 1]
```diff
@@ -43,7 +44,6 @@ class ApiWrapper
     // Prince Options
     protected $baseurl;
```
Comment (janxious) [inline src/ApiWrapper.php]: Consider moving `$baseurl` up to the "Output document related" section.

File: src/ApiWrapper.php [hunk 2]
```diff
@@ -264,6 +278,12 @@ public function fetchDocument($filename = false)
             $fields['doc[prince_options][baseurl]'] = $this->baseurl;
         }
 
+        if (!empty($this->document_prince_options) && is_array($this->document_prince_options)) {
```
Comment (janxious) [inline src/ApiWrapper.php]: I don't think you need the `!empty` check here. Also consider moving the array validation to `ApiWrapper.php`. `ApiWrapper` is responsible for making sure the data is valid. This class is for making sure the request is properly formatted.

**Comment by janxious** [comment]
@rsands2801 Let me know if you want me to make my proposed changes. Can get this out and published today, probably.

**Comment by rsands2801** [comment]
I have move baseurl to the document options part.  I suppose this variable should disappear as it can be part of the setdocumentprinceoptions but to leave simpler functionality i didn't.

For the is_array check do you think we could just remove this completely?

**Pull request merged**